src/shell/casparcg.config

```diff
@@ -135,6 +138,30 @@
                 <path>[file|url]</path>
                 <args>[most ffmpeg arguments related to filtering and output codecs]</args>
             </ffmpeg>
+            <artnet>
+                <universe>0</universe>
+
+                <host>127.0.0.1</host>
+                <port>6454</port>
+
+                <refresh-rate>30</refresh-rate>
+
+                <fixtures>
+                    <fixture>
+                        <type>RGB</type>
+                        <start-address>1</start-address>
+                        <fixture-count>10</fixture-count>
```

Comment:
I don't know _too much_ about lighting, but would it be useful to have some stride parameters here? If I'm not mistaken lots of fixtures have have other parameters besides RGBW that we don't want to control from CasparCG.

Reply:
Yeah, you are right. I am going to fix a stride setting to create an offset between fixtures.